compute/cker/include/cker/CpuBackendThreadpool.h

```diff
@@ -33,8 +35,16 @@ using Task = ruy::Task;
 template <typename TaskType>
 void Execute(int tasks_count, TaskType *tasks, ruy::Context *ruy_context)
 {
+  assert(ruy_context != nullptr);
   assert(tasks_count <= ruy_context->max_num_threads());
-  ruy_context->mutable_thread_pool()->Execute(tasks_count, tasks);
+  if (ruy_context != nullptr)
+  {
+    ruy_context->mutable_thread_pool()->Execute(tasks_count, tasks);
+  }
+  else
+  {
+    throw std::runtime_error("CpuBackendThreadpool.h: ruy::Context is null");
+  }
```

Comment:
```suggestion
  if (ruy_context == nullptr)
  {
    throw std::runtime_error("CpuBackendThreadpool.h: ruy::Context is null");
  }
  ruy_context->mutable_thread_pool()->Execute(tasks_count, tasks);
```

We can remove `else` if we throw exception first.